test/lib/fhirValidatorTest.ts

```diff
@@ -393,68 +393,60 @@ describe('fhir validator', () => {
       expect(valid).to.equal(false);
     });
 
-    it('fails to validate with an explicit message if no parameter is provided', done => {
+    it('fails to validate with an explicit message if no parameter is provided', async () => {
       try {
         // @ts-ignore
-        fhirValidator.validate();
-        done();
+        await fhirValidator.validate();
```

Comment:
I assume that this would also run through when no error is thrown.
I see two easy fixes: either an `expect(true).to.be.false` in the try block.
Or something like:
let error;
`try {} catch (err){ error = err} expect(err).to.not.be.null...`